lib/services/users.pm

```diff
@@ -193,4 +193,19 @@ sub full_users_check {
     }
 }
 
+# cleanup for exceptions drung before and after migration
+sub users_cleanup {
+    my (%hash) = @_;
+    my $stage = $hash{stage};
+    select_console "root-console";
+    if ($stage eq 'before') {
+        script_run('pkill -KILL -u test');
```

Comment:
Please use the variable for the user 'test',  $newUser to avoid the hardcode.

Reply:
OK